src/components/Program/Program.jsx

```diff
@@ -0,0 +1,91 @@
+// Program.js
+import {useState, useEffect} from 'react'
+import Section from '../Section/Section'
+import css from './Program.module.css'
+import SingleSchedule from './SingleSchedule'
+
+const Program = () => {
+  const [isMobileView, setIsMobileView] = useState(window.innerWidth < 756)
+  const [expandedDetails, setExpandedDetails] = useState(Array(2).fill(false))
+
+  useEffect(() => {
+    const handleResize = () => {
+      setIsMobileView(window.innerWidth < 756)
+    }
+
+    window.addEventListener('resize', handleResize)
+
+    return () => {
+      window.removeEventListener('resize', handleResize)
+    }
+  }, [])
+
+  const toggleDetails = (index) => {
+    const newExpandedDetails = [...expandedDetails]
+    newExpandedDetails[index] = !newExpandedDetails[index]
+    setExpandedDetails(newExpandedDetails)
+  }
```

Comment:
możesz mi powiedzieć do czego są te stany? Bo nie za bardzo ogarniam ich funkcję

Reply:
układ jest inny dla rozdzielczości do 765px  - godzina - svg i poniżej tekst, a powyżej 765px godzina-tekst i svg.  handleResize nasłuchuje jaka jest rozdzielczość i wyświetla odpowiedni układ (aktualizuje stan isMobile) w zależności od szerokości okna a potem go odmontowuje. 
toggleDetails zmienia stan rozwinięcia szczegółów dla elementu o danym indexie. Tworzy nową kopię tablicy expandedDetails (newExpandedDetails), zmienia wartość dla określonego indeksu na przeciwną (za pomocą !), a następnie aktualizuje stan expandedDetails nową tablicą.

Reply:
A nie łatwiej ten svg było wstawić na absolute i wypozycjonować w top/right a na pc wycentrować?

Reply:
Super to wymyśliłaś z tym nasłuchem ale myślę że prościej było ten ptaszek w position absolute dać i ustawić tak aby w mobilce był wyżej a na tab/pc wyśrodkowany

Reply:
najprostszy sposób - najlepszy :)